HousingFinanceInterimApi.Tests/V1/Infrastructure/DatabaseContext/Reports/GetCashImportByDateTests.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoFixture;
+using HousingFinanceInterimApi.Tests.V1.TestHelpers;
+using HousingFinanceInterimApi.V1.Gateways;
+using HousingFinanceInterimApi.V1.Infrastructure;
+using Xunit;
+
+namespace HousingFinanceInterimApi.Tests.V1.Infrastructure.DatabaseContext.Reports;
+
+public class GetCashImportByDateTests : IClassFixture<BaseContextTest>
+{
+    private readonly HousingFinanceInterimApi.V1.Infrastructure.DatabaseContext _context;
+    private readonly Fixture _fixture;
+
+    public GetCashImportByDateTests(BaseContextTest baseContextTest)
+    {
+        _context = baseContextTest._context;
+        var cleanups = baseContextTest._cleanups;
+        _fixture = new Fixture();
+
+        _fixture.Customize<UPCashDumpFileName>(composer => composer
+            .Without(x => x.Id)
+            .Without(x => x.Timestamp)
+        );
+
+        var tablesToClear = new List<Type>
+        {
+            typeof(SSMiniTransaction), typeof(UPCashDumpFileName), typeof(UPCashDump), typeof(UPCashLoad)
+        };
+        ClearTable.ClearTables(_context, tablesToClear);
+        cleanups.Add(() => ClearTable.ClearTables(_context, tablesToClear));
```

Comment:
If every tests fixture ends up adding 'clear all tables before you start & after every test.
It may as well be the default behaviour of the base context tests Dispose logic.

Reply:
Different tests would need to clear different tables though. I think clearing all tables before and after each test case would be a significant overhead, not sure though